modules/client/src/main/java/org/apache/ignite/internal/client/io/netty/NettyClientConnectionMultiplexer.java

```diff
@@ -75,6 +94,77 @@ public void initChannel(SocketChannel ch) {
         }
     }
 
+    private void setupSsl(SocketChannel ch, IgniteClientConfiguration clientCfg) {
+        if (clientCfg.sslConfiguration() == null || !clientCfg.sslConfiguration().enabled()) {
+            return;
+        }
+
+        try {
+            var ssl = clientCfg.sslConfiguration();
```

Comment:
These usages of `var` contradict the code style

Reply:
Fixex these two. Does it relate to `try (var is = Files.newInputStream(Path.of(ssl.keyStorePath())))`?

Reply:
ok, will fix that too